### ISSUE TITLE
Slim down `evaluate` implementation

### ISSUE THREAD

**Issue by dtch1997** [dtch1997/repepo#156]
- TokenProbs
  - Remove saving logits
- EvalPrediction
  - Remove TextProbs
- `Pipeline.calculate_output_logprobs`
  - Remove logit statistics
